pyproject.toml

```diff
@@ -9,7 +9,8 @@ packages = [
 include = ["config/*.json"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+#python = "^3.6"
```

Comment:
We don't need to preserve the old value here.

```suggestion
#python = "^3.6"
```

Reply:
In general we avoid commented out code. This can always be restored from version control.

Reply:
Thank you I will make a note of this for the future!